application/helpers/pdf_helper.php

```diff
@@ -75,7 +75,7 @@ function generate_invoice_pdf($invoice_id, $stream = true, $invoice_template = n
     $html = $CI->load->view('invoice_templates/pdf/' . $invoice_template, $data, true);
 
     $CI->load->helper('mpdf');
-    return pdf_create($html, trans('invoice') . '_' . str_replace(array('\\', '/'), '_', $invoice->invoice_number),
+    return pdf_create($html, lang('invoice') . '_' . str_replace(array('\\', '/'), '_', $invoice->invoice_number . '_' . $invoice->invoice_date_created . '_' . $invoice->client_name),
```

Comment:
Why change the trans() to lang()? I believe trans is needed for localisation.
The rest looks good.